js/renderer/texturesprite_renderer.js

```diff
@@ -26,6 +27,44 @@ LayerScope.SpriteView = {
     $("#texture-container").animate({scrollTop:top}, '500', 'swing');
   },
 
+  init: function TR_init(graph) {
+    $('#texture-property-table').dataTable({
+      "bScrollInfinite": true,
+      "bScrollCollapse": true,
+      "sScrollY": "210px",
+      "scrollCollapse": true,
+      "paging":         false,
+      "columns": [ {"width": "40%"}, {"width": "60%"} ]
+    });
+    LayerScope.MessageCenter.subscribe("texture.select", this);
+    LayerScope.MessageCenter.subscribe("color.select", this);
+  },
+
+  begin: function LR_begin() {
+    // Clear is not enough, we need to force draw here.
+    $("#texture-property-table").DataTable().clear();
+    $("#texture-property-table").DataTable().draw();
+  },
+
+  notify: function LR_notify(name, texNode) {
+    var $table = $("#texture-property-table").DataTable();
+    $table.clear();
+    if (name === "texture.select") {
+      if(!!texNode) {
+        var dataSet = [];
+        generateTextureAttributes(texNode, dataSet);
+        if (dataSet.length > 0) {
+          for (let i = 0; i< dataSet.length; i++) {
+            $table.row.add(dataSet[i]);
+          }
+        }
+      }
+    } else if (name === "color.select") {
+      // TODO: Generate color attributes here
+    }
+    $table.draw();
```

Comment:
I don't suggest always draw here. Please consider move this call into if/else if/else statements